stages/org.osbuild.pki.update-ca-trust

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+import os
+import sys
+import uuid
+
+import osbuild.api
+
+
+def main(tree, options):
+    for cert in options["certs"]:
+        rnd = str(uuid.uuid4())
+        with open(f"{tree}/etc/pki/ca-trust/source/anchors/ostree-{rnd}.crt", "w", encoding="utf8") as f:
+            f.write(cert)
+
+    subprocess.run(["/usr/bin/update-ca-trust", "extract"], check=True)
+    return 0
```

Comment:
(nitpick) I think we can just remove this the "sys.exit(r)" below. We never return anything but "0" here and any issue will be raised via an exception which automatically triggers a non-zero exit code :)